src/components/ValueTypeDropDown.vue

```diff
@@ -57,12 +72,7 @@ export default Vue.extend( {
 			width: 256px;
 			height: 32px;
 			margin-inline-start: $dimension-layout-xsmall;
-			margin-block-start: 23px;
-			border-color: #a2a9b1;
-			border-style: solid;
-			border-width: 1px;
-			border-radius: 2px;
-			appearance: auto;
+			margin-block-start: 21px;
```

Comment:
shouldn't all this style be removed ? 
AFAIK, the styles are already implemented in the DS. 
just like in the lookup component

Reply:
I removed most of them, but If I remove the rest, it'll look like this:
![image](https://user-images.githubusercontent.com/5351225/101170374-a63fbf80-363e-11eb-936f-c8949e915226.png)

Reply:
hmm... @micgro42 could you verify that we don't have any style defined for the lookup component in the QB ?
if so, should it have been the same case with the dropdown ?
thanks

Reply:
will have a look

Reply:
I moved the responsibility of arranging the child components to the QueryCondition component